docs/en/ingest-management/elastic-agent/ingest-pipeline-in-Kubernetes.asciidoc

```diff
@@ -0,0 +1,83 @@
+[[ingest-pipeline-in-Kubernetes]]
+= Using Custom Ingest Pipeline with {k8s} Integration
+
+This tutorial explains how to add a custom ingest pipeline to {k8s} Integration in order to add specific metadata fields for deployments and cronjobs of pods.
+
+Custom pipelines can be used to add custom data processing, like adding fields, obfuscate sensitive information, and more. Find more information in [data-streams-pipeline-tutorial](https://www.elastic.co/guide/en/fleet/master/data-streams-pipeline-tutorial.html) on how to use custom ingest pipelines.
+
+== Metadata Enrichment for Kubernetes
+
+The https://docs.elastic.co/en/integrations/kubernetes[{k8s} Integration] is used to collect logs and metrics from Kubernetes clusters with {agent}. During the collection, integration enhances the collected information with extra useful information that can help users to correlate information and different Kubernetes assets with each other. All this additional information that are added on top of collected information like labels, annotations, ancestor names of Kubernetes assets etc. are called metadata.
```

Comment:
```suggestion
The https://docs.elastic.co/en/integrations/kubernetes[{k8s} Integration] is used to collect logs and metrics from Kubernetes clusters with {agent}. During the collection, the integration enhances the collected information with extra useful information that users can correlate with different Kubernetes assets. This additional information added on top of collected data, such as labels, annotations, ancestor names of Kubernetes assets, and others, are called metadata.
```
Just rephrasing slightly since `information` appears several times.